tests/test_interface_unversioned.py

```diff
@@ -91,6 +91,7 @@ def tree(tmpdir, request):
     [
         (audbackend.backend.Artifactory, audbackend.interface.Unversioned),
         (audbackend.backend.FileSystem, audbackend.interface.Unversioned),
+        (audbackend.backend.Minio, audbackend.interface.Unversioned),
```

Comment:
The interface parametrization seems to vary little between tests. 

Would it make sense to recast this into a fixture?  Possibly like this:


```python
@pytest.fixture(params=[
    (audbackend.backend.Artifactory, audbackend.interface.Unversioned),
    (audbackend.backend.FileSystem, audbackend.interface.Unversioned),
    (audbackend.backend.Minio, audbackend.interface.Unversioned),
    (SingleFolder, audbackend.interface.Unversioned),
])
def interface(request):
    return request.param
```

Reply:
`interface` was already a fixture, but I could solve the problem by a variable:

```python
# Backend-interface combinations to use in all tests
backend_interface_combinations = [ 
    (audbackend.backend.Artifactory, audbackend.interface.Unversioned),
    (audbackend.backend.FileSystem, audbackend.interface.Unversioned),
    (audbackend.backend.Minio, audbackend.interface.Unversioned),
    (SingleFolder, audbackend.interface.Unversioned),
]

# ...

@pytest.mark.parametrize(
    "interface",
    backend_interface_combinations,
    indirect=True,
)
def test_archive(tmpdir, tree, archive, files, tmp_root, interface, expected):
```